### DISCUSSION TITLE
My github page wont load

### DISCUSSION THREAD

**Discussion by Plazl** [community/community#23882] [Other Feature Feedback, Questions, & Ideas]
<p>I am making a basic clicker game using github pages but for some reason the page 404s, i have linked everything but it still wont work. here is a screenshot of my page settings (all of the code is in the main branch) </p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/0/5/058060b318ae58e23a5621d4e7c9aee4d555a4d4.png" data-download-href="/uploads/short-url/MFqZAlKeKzUm8Nhnj8v7GfRnG4.png?dl=1" title="Screenshot 2021-09-01 2.19.26 PM" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/86067301/181079825-8ae15ef9-bb83-46f5-bac2-2868600d0fce.png" alt="Screenshot 2021-09-01 2.19.26 PM" data-base62-sha1="MFqZAlKeKzUm8Nhnj8v7GfRnG4" width="690" height="321" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/0/5/058060b318ae58e23a5621d4e7c9aee4d555a4d4_2_690x321.png, https://aws1.discourse-cdn.com/github/optimized/3X/0/5/058060b318ae58e23a5621d4e7c9aee4d555a4d4_2_1035x481.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/0/5/058060b318ae58e23a5621d4e7c9aee4d555a4d4.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/0/5/058060b318ae58e23a5621d4e7c9aee4d555a4d4_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot 2021-09-01 2.19.26 PM</span><span class="informations">1365×636 75.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div><br>
here is the repository <a href="https://github.com/Plazl/Evo-Clicker" class="inline-onebox" rel="noopener nofollow ugc">GitHub - Plazl/Evo-Clicker</a> and here is the page link <a href="https://plazl.github.io/Evo-Clicker/" rel="noopener nofollow ugc">https://plazl.github.io/Evo-Clicker/</a><br>
something that might be causing this is the amount of inline js that i used

**Comment by jfullstackdev** [comment]
<p>I tested your GitHub Page site, but I cannot access the LICENSE file. If the GitHub Page is working correctly, all the files can be accessed, including the LICENSE file. Meaning, you don’t have the page, it’s really 404.</p>
<p>Did you click the <code>Save</code> button? Make sure you clicked that, if not try to build the page again.</p>

**Comment by Plazl** [comment]
<p>i removed the pages environment then set it back up but now the page wont even load (the page is just black) so is this due to inline js?</p>

**Comment by jfullstackdev** [comment]
<p><a href="https://plazl.github.io/Evo-Clicker/LICENSE" rel="noopener nofollow ugc">https://plazl.github.io/Evo-Clicker/LICENSE</a></p>
<p>this is the test, the license was successfully downloaded, meaning, you now have the page,</p>
<p>now remove the <code>index.md</code> first, then try to put some words on your README, you should see that on your page, meaning your page is working, wait more than a minute to test</p>

**Comment by jfullstackdev** [comment]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/8/3/83dd0b48406f88d9ffb4485141ac19d054ba39d9.jpeg" data-download-href="/uploads/short-url/iOw8gF9DnghLAON6n70rMhaOEUV.jpeg?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/47092464/181079829-24e2605c-dbae-4781-85ce-cecf40a64f41.jpeg" alt="image" data-base62-sha1="iOw8gF9DnghLAON6n70rMhaOEUV" width="690" height="388" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/8/3/83dd0b48406f88d9ffb4485141ac19d054ba39d9_2_690x388.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/8/3/83dd0b48406f88d9ffb4485141ac19d054ba39d9_2_1035x582.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/8/3/83dd0b48406f88d9ffb4485141ac19d054ba39d9_2_1380x776.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/8/3/83dd0b48406f88d9ffb4485141ac19d054ba39d9_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1920×1080 50.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>now, you have the page,</p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>now, put all of your site files, if that is not working, it’s your code that is not working, maybe the loading of the JavaScript, because a while ago, it’s just loading and loading, so it’s now your turn to detect the mistakes at your code,</p>

**Comment by Plazl** [comment]
<p>thank you for the help!</p>
